example/config/development.json

```diff
@@ -1 +1 @@
-{"warp": { "port": 9999}, "a": 7}
+{"server": { "port": 9999}, "a": 7}
```

Comment:
I see now that the example is wrong, but I'd like this PR to only touch what it says it will touch, can you remove these changes? if you want to create a different PR that'd be nice, but I'll do it myself in the future anyhow.

Reply:
reverted